src/views/Login/PersonaRegistration/TermsConditions/index.tsx

```diff
@@ -0,0 +1,170 @@
+          target="_blank"
+          rel="noreferrer"
+        >
+          NIH Security Best Practices for Controlled-Access Data
+        </a>{' '}
+        such that only authorized individuals possess access to data files;
+      </li>
+      <li>
+        You acknowledge Intellectual Property Policies should they exist as specified in a dataset's
+        associated Data Use Certification; and,
+      </li>
+      <li>
+        {' '}
+        You will report any inadvertent data release in accordance with the terms in the Data Use
+        Certification, breach of data security, or other data management incidents contrary to the
+        terms of data access.
+      </li>
+    </ul>
+    <div>
+      DRC terms and conditions may be changed at any time via a public posting of revisions to the
+      Services. As a user, you agree to review the{' '}
+      <a href="https://kidsfirstdrc.org/policies/" target="_blank" rel="noreferrer">
+        Terms & Conditions and Privacy Policies
+      </a>{' '}
+      each time you use the Services so that you are aware of any modifications made to these
+      policies. By accessing or using the Services, you agree with and to be bound by all of the
+      terms and conditions and policies as posted on the Services at the time of your access or use,
+      including the Privacy Policies then in effect.
+    </div>
+    <div>
+      For documents and/or data available from the Services, the DRC does not warrant or assume any
+      legal liability or responsibility for the accuracy, completeness, or usefulness of any
+      information, apparatus, product, or process. No specific medical advice is provided by any
+      Services, and the Kids First DRC urges users of Services to consult with a qualified physician
+      for diagnosis and for answers to personal questions.
+    </div>
+    <div>
+      For more information about the NIH Genomic Data Sharing Policy and applying for
+      Controlled-Access Data through dbGaP visit:
+    </div>
+    <ul>
+      <li>
+        NIH Genomic Data Sharing Policy:
+        <a href="https://grants.nih.gov/grants/guide/notice-files/NOT-OD-14-124.html">
+          https://grants.nih.gov/grants/guide/notice-files/NOT-OD-14-124.html
+        </a>{' '}
+        dbGaP Request Procedures to Access Individual-Level Data:
+        <a href="https://dbgap.ncbi.nlm.nih.gov/aa/dbgap_request_process.pdf">
+          https://dbgap.ncbi.nlm.nih.gov/aa/dbgap_request_process.pdf
+        </a>
+      </li>
+    </ul>
+    <div>
+      If you have any questions about these terms, conditions or the practices of this site or any
+      of the other Services, please contact us at
+      <a href="mailto:support@kidsfirstdrc.org">support@kidsfirstdrc.org</a>.
+    </div>
+  </Space>
+);
+
+const TermsConditions = ({ hidden, onFinish }: OwnProps) => {
+  const history = useHistory();
```

Comment:
seems like a bit of a duplication with the registration form. Think about using a custom hook or anything else if you think it s worth it.

Reply:
For two lines, I don't think it worth it